packages/webdriverio/webdriverio.d.ts

```diff
@@ -3,22 +3,33 @@
 type BrowserObject = WebDriver.ClientOptions & WebDriver.ClientAsync & WebdriverIOAsync.Browser;
 
 // Element commands that should be wrapper with Promise
-type ElementPromise = Omit<WebdriverIO.Element, 'addCommand'>;
+type ElementPromise = Omit<WebdriverIO.Element, 'addCommand' | '$' | '$$'>;
+
+// Methods which return async element(s) so non-async equivalents cannot just be promise-wrapped
+interface AsyncSelectors {
+    $(
```

Comment:
small suggestion, can we move $ and $$ to types and reuse types in global variables declarations, something like this:
```
type $ = (selector: string | Function) => Promise<WebdriverIOAsync.Element>;
type $$ = (selector: string | Function) => Promise<WebdriverIOAsync.Element[]>;
interface AsyncSelectors {
    $: $;
    $$: $$;
}
```
...
```
declare var browser: BrowserObject;
declare var $: $;
declare var $$: $$;
```

What do you think?

Reply:
I can see the point of the types, but for my benefit could you explain why these should be declared globally? Are they supposed to be accessible that way?

Reply:
@richsilv if you use the wdio testrunner yes

Reply:
Makes sense, thanks! I'll make the requested change.

Reply:
okay, done